build/Build.cs

```diff
@@ -220,22 +247,37 @@ class Build : NukeBuild
         }
     });
 
+    Target PackNormalClientNugetForMacOs => _ => _
+      .OnlyWhenStatic(() => IsLocalBuild)
+      .OnlyWhenStatic(() => OperatingSystem.IsMacOS())
+      .DependsOn(Compile)
+      .Executes(() =>
+      {
+          Log.Warning("Building an Unsigned Normal Client Nuget Package for MacOS");
+
+          PackNormalClientNugetPackage();
+      });
+
     Target PackNormalClientNuget => _ => _
         .DependsOn(Compile)
         .Executes(() =>
     {
         SignBinaries(OctopusNormalClientFolder / "bin" / Configuration);
 
-        DotNetPack(_ => _
-            .SetProject(OctopusNormalClientFolder)
-            .SetVersion(FullSemVer)
-            .SetConfiguration(Configuration)
-            .SetOutputDirectory(ArtifactsDir)
-            .EnableNoBuild()
-            .DisableIncludeSymbols()
-            .SetVerbosity(DotNetVerbosity.Normal));
+        PackNormalClientNugetPackage();
     });
 
+    void PackNormalClientNugetPackage()
```

Comment:
Should this be it's own Nuke target rather than a function, so other targets can DependsOn it?

Reply:
I don't think we can in this case because of the order of execution. For the Signed Client we need to sign the binaries before packing which means the packing target would need to depend on the signing target but then we can't use the packing target for anything else because you can't have a conditional dependency which is a shame.